### PR TITLE
add presubmit_build.sh that passes a flag to swiftpm if it is a stock toolchain

### PR DESCRIPTION
**Pull request by marcrasi**
I copied this from the internal script in google3, and modified it to pass a flag when building with a stock toolchain.

I'll also make a CL that deletes the script from google3 and calls this one instead.

### CODE REVIEW

File: docker/presubmit_build.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Help debug the job's disk space.
+df -h
+
+cd github/swift-models
+sudo -E docker build -t built-img -f docker/Dockerfile --build-arg swift_tf_url .
+
+# SwiftPM-based build.
+docker run built-img /bin/bash -c "
+swift build ${STOCK_TOOLCHAIN:+"-D -Xswiftc -D TENSORFLOW_USE_STANDARD_TOOLCHAIN"} ;
+swift test ${STOCK_TOOLCHAIN:+"-D -Xswiftc -D TENSORFLOW_USE_STANDARD_TOOLCHAIN"} ;
```
Comment (BradLarson) [inline docker/presubmit_build.sh:35]: I think there might be an extra -D in both of these. Should it just be `"-Xswiftc -D TENSORFLOW_USE_STANDARD_TOOLCHAIN"`?

Comment (marcrasi) [reply docker/presubmit_build.sh]: Good catch. And there should also be another `-Xswiftc`, right?

Comment (BradLarson) [reply docker/presubmit_build.sh]: I think just the one is needed if you smoosh the last two together into `"-Xswiftc -DTENSORFLOW_USE_STANDARD_TOOLCHAIN"`. That works for me at the command line.

Comment (BradLarson) [reply docker/presubmit_build.sh]: But I tried what you have now with the two of them, and that also works well.

**Review by BradLarson — APPROVED** [review]

**Pull request merged**